internal/engine/ydb/convert.go

```diff
@@ -869,30 +993,30 @@ func (c *cc) convertUpdate_stmtContext(n *parser.Update_stmtContext) ast.Node {
 	return stmts
 }
 
-func (c *cc) convertInto_table_stmtContext(n *parser.Into_table_stmtContext) ast.Node {
-	tableName := identifier(n.Into_simple_table_ref().Simple_table_ref().Simple_table_ref_core().GetText())
+func (c *cc) VisitInto_table_stmt(ctx *parser.Into_table_stmtContext) interface{} {
```

Comment:
`ctx` используется для `context.Context`. Лучше другое имя аргумента. Был `n` - в целом тоже норм. И diff будет меньше.

```suggestion
func (c *cc) VisitInto_table_stmt(n *parser.Into_table_stmtContext) interface{} {
```